### PR TITLE
Added new javascript tutorial link

### LINKED ISSUE #1942: [Add] <Want to add youtube playlist of Javascript in hindi>
### Site URL

https://youtube.com/playlist?list=PLwgFb6VsUj_n15Cg_y2ULKfsOR1XiQqPx&si=UJyXUgEH0_94SpYj

### Description of the above provided site

Complete javascript from beginner to mastery.

### Category

Languages

### Sub Category

JavascriptInHindi

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Atul72**
##Proposed Changes
Add new javascript link

## Fixes Issue
Closes #1942

## Screenshots

I did't set it up locally, just add the link. 

## Note to reviewers
Kindly add this PR.

### CODE REVIEW

File: database/languages/javascript.json [hunk 1]
```diff
@@ -75,5 +75,12 @@
...
     "category": "languages",
     "subcategory": "javascript"
+  },
+  {
+    "name": "javascript Beginning To Mastery",
+    "description": "This playlist makes you a advance javascript programmer. Teach everything from beginning to advance level in Hindi.",
+    "url": "https://youtube.com/playlist?list=PLwgFb6VsUj_n15Cg_y2ULKfsOR1XiQqPx&si=UJyXUgEH0_94SpYj",
+    "category": "languages",
+    "subcategory": "javascript"
   }
```
Comment (CBID2) [inline database/languages/javascript.json:85]: [proofreading] As mentioned in the guidelines [here](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md#appending-new-links-), you need to add a `language` field in your source is not in English.  
```suggestion
    "name": "javascript Beginning To Mastery",
    "description": "This playlist makes you a advance javascript programmer. Teach everything from beginning to advance level in Hindi.",
    "url": "https://youtube.com/playlist?list=PLwgFb6VsUj_n15Cg_y2ULKfsOR1XiQqPx&si=UJyXUgEH0_94SpYj",
    "category": "languages",
    "subcategory": "javascript",
    "language": "hindi"
  }
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Atul72! :) You're off to a great start! :) You forgot to add some fields and there were a few punctuation errors, so refer to my inline comment to help you. In the future, I highly recommend reading the guidelines in our Contributing Guide to format your resources and running your descriptions of them through Grammarly before submitting them as PRs.

**Review by CBID2 — APPROVED** [review]
Well done @Atul72! :) @rupali-codes and @Anmol-Baranwal, you're up! :)

**Review by CBID2 — APPROVED** [review]
:)

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by rupali-codes — APPROVED** [review]
All good!

**Pull request merged**